File: lingting-security/src/main/java/live/lingting/framework/security/exception/AuthorizationException.java
package live.lingting.framework.security.exception;

/**
 * @author lingting 2023-03-29 21:41
 */
public class AuthorizationException extends SecurityException {

	public AuthorizationException() {
		super();
	}

	public AuthorizationException(String message) {
		super(message);
	}

	public AuthorizationException(String message, Throwable cause) {
		super(message, cause);
	}

}

File: lingting-security/src/main/kotlin/live/lingting/framework/security/exception/AuthorizationException.kt
package live.lingting.framework.security.exception

/**
 * @author lingting 2023-03-29 21:41
 */
class AuthorizationException : SecurityException {
    constructor() : super()

    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)
}
